src/core/iomgr/pollset_multipoller_with_epoll.c

```diff
@@ -180,7 +181,9 @@ static void multipoll_with_epoll_pollset_maybe_work(
   pfds[1].events = POLLIN;
   pfds[1].revents = 0;
 
+  GRPC_IOMGR_START_BLOCKING_REGION;
```

Comment:
It may be worthwhile (although probably not in the initial version of this PR) to do an unguarded 0 timeout poll here.